documentation/api/io.strimzi.api.kafka.model.KafkaMirrorMaker2ClusterSpec.adoc

```diff
@@ -0,0 +1,18 @@
+Use the three allowed `ssl` configuration options to run external listeners on a specific _cipher suite_ for a TLS version.
+A _cipher suite_ combines algorithms for secure connection and data transfer.
+
+.Example SSL configuration
+[source,yaml,subs="attributes+"]
+----
+spec:
+  clusters:
+    config:
+      ssl.cipher.suites: "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384" <1>
+      ssl.enabled.protocols: "TLSv1.2" <2>
+      ssl.protocol: "TLSv1.2" <3>
+----
+<1> The cipher suite for TLS using a combination of `ECDHE` key exchange mechanism, `RSA` authentication algorithm,
+`AES` bulk encyption algorithm and `SHA384` MAC algorithm.
+<2> The SSl protocol `TLSv1.2` is enabled.
+<3> Specifies the `TLSv1.2` protocol to generate the SSL context.
+Allowed values are `TLSv1.2` and `TLSv1.2`.
```

Comment:
`TLSv1.2` and `TLSv1.2` ? Same question for all the other files.

Reply:
Allowed values are `TLSv1`, `TLSv1.1` and `TLSv1.2`.